DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
Please remove any appearance of `trackIp2dVal` and `trackIp2dSig` in `DataFormats/BTauReco/interface/TaggingVariable.h` and `DataFormats/BTauReco/src/TaggingVariable.cc`. These are not used anywhere so better to remove them.